ChartIQ/Charts/ChartIQView.swift

```diff
@@ -1334,6 +1338,21 @@ extension ChartIQView: WKScriptMessageHandler {
                     UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, quote);
                 }
             }
+        // Allows for various console messages from JavaScript to show up in Xcode console.
```

Comment:
I would put the comment inside of the case, but follow how it is done in the rest of the project.

Reply:
Agree, I'll move it into the case.  There aren't other comments like this in the code, so I'll start the trend of having it inside.